crowd_anki/config/config_settings.py

```diff
@@ -0,0 +1,88 @@
+from collections import namedtuple
+from enum import Enum
+
+from aqt import mw
+from ..utils.constants import USER_FILES_PATH
+
+ConfigEntry = namedtuple("ConfigEntry", ["config_name", "default_value"])
+
+
+class NoteSortingMethods(Enum):
+    FIELD1 = "field1"
+    FIELD2 = "field2"
+    NO_SORTING = "none"
```

Comment:
I think the name value should match here to, not sure which way though 😅

Reply:
If you feel it's alright for the user to type "no_sorting" then simply say so and I shall change it! :100: :+1:  Especially with the helpful text at the side, and a warning when an incorrect value is entered, it should be no issue. I simply dislike such entry systems myself, and thus do not wish to make one like that :smile: but we cannot all have what we want. Final call is yours :+1:

Reply:
eh, I don't have strong arguments either way on this one, the `notemodelname` and such should definitely haven underscores in it thouhg ;)

Reply:
Alright, I'll leave `none` as it is, and change the others to `note_model_name` and `note_model_id` :+1: